src/Discord.Net.Rest/Entities/Channels/ChannelHelper.cs

```diff
@@ -76,15 +76,16 @@ internal static class ChannelHelper
         public static async Task<RestInviteMetadata> CreateInviteAsync(IGuildChannel channel, BaseDiscordClient client,
             int? maxAge, int? maxUses, bool isTemporary, bool isUnique, RequestOptions options)
         {
-            var args = new CreateChannelInviteParams { IsTemporary = isTemporary, IsUnique = isUnique };
-            if (maxAge.HasValue)
-                args.MaxAge = maxAge.Value;
-            else
-                args.MaxAge = 0;
-            if (maxUses.HasValue)
-                args.MaxUses = maxUses.Value;
-            else
-                args.MaxUses = 0;
+            if (maxAge > 86400)
+                throw new ArgumentOutOfRangeException(nameof(maxAge), maxAge,
+                    "The maximum age of an invite must be less than or equal to a day (86400).");
```

Comment:
What do you think about the wording:
"... less than or equal to a day's length in seconds (86400)."

So that it can include units?

Reply:
I'd say `less than or equal to a day (86400 seconds)`

Reply:
I think that reads better.